src/content/en/2022/fonts.md

```diff
@@ -1,22 +1,851 @@
+
+To us, it appears that variable fonts are primarily used for their performance benefits, and less so for their ability to make typographic adjustments. While that's great, we do hope to see more people use variable fonts to their full typographic potential in the coming years.
+
+## Color fonts
+
+Color fonts are pretty much what you would expect: fonts with built in colors. Though the technology was originally created for emoji fonts, there are now more text color fonts than emoji fonts.
+
+{{ figure_markup(
+  image="color-font-usage.png",
+  caption="Color font usage.",
+  description="Column chart showing that in 2020 color fonts were used on 0.005% of desktop pages and 0.004% of mobile pages, 2021 on 0.013% and 0.015% respectively, 2022 on 0.015% of desktop and 0.018% of mobile pages..",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vQF6OH_2-0apcFzjE-iHSQNuZqp9DtM7udIeOzPSOSMM-Pf6KdTnRwAclX9QPZF1vNNgu6acZvqoN5b/pubchart?oid=1366906842&format=interactive",
+  sheets_gid="1630630917",
+  sql_file="color_font_usage.sql"
+  )
+}}
+
+Color fonts usage has grown quite a bit since the last Fonts chapter in 2020. Usage went from 0.004% of pages using color fonts in 2020 to about 0.018% in 2022. While those numbers are still very small, there is a clear growth in their usage.
+
+However, compared to the growth in variable font usage, the limited uptake of color fonts is somewhat disappointing. While color fonts are a relatively new addition to the OpenType specification (2015), variable fonts are an even more recent addition (2016).
+
+The primary factors that have severely hindered color font adoption (and might continue to do so) is the ongoing standards "battle" for _the one true color font format_, and the lack of support in browsers for the CSS that allows you to select and edit color font palettes—until recently.
+
+There are currently four competing color font formats: two based on vector outlines (OT-SVG and COLR) and two on images (CBDT and sbix). The COLR format re-uses the existing glyph outline and adds solid colors and layering to them. The most recent version, dubbed COLRv1 introduced gradients, compositing and blending modes as well. Due to its re-use of existing glyph outlines, the COLR format also supports variable fonts, so you can have <a hreflang="en" href="https://www.typearture.com/variable-fonts/">animated color fonts</a>. The OT-SVG format takes a different approach and essentially embeds an SVG image for each glyph in the font. Unfortunately, the OT-SVG format does not support variable fonts, and is unlikely to do so in the future. Both CBDT and sbix embed images for each glyph and they only differ in the supported image formats.
```

Comment:
Should OT-SVG, COLR, CBDT and sbix be in code format? OT-SVG as earlier.

Reply:
OT-SVG is shorthand for the OpenType SVG format so it is just an acronym (primarily used so there is no confusion with "regular" SVG, or the old SVG font format). COLR, CBDT and sbix on the other hand are OpenType table tags and should be formatted as such.